nodejs/week3/app.js

```diff
@@ -0,0 +1,49 @@
+const knex = require("knex")({
+  client: "mysql2",
+  connection: {
+    host: process.env.DB_HOST || "127.0.0.1",
+    port: process.env.DB_PORT || 3306,
+    user: process.env.DB_USER || "root",
+    password: process.env.DB_PASSWORD || "1237895hamodi",
+    database: process.env.DB_NAME || "hyf_node_week3_warmup",
+    multipleStatements: true,
+  },
+});
+
+const express = require("express");
+const app = express();
+const port = process.env.PORT || 3000;
+
+app.use(express.json());
+
+const apiRouter = express.Router();
+app.use("/api", apiRouter);
+
+const contactsAPIRouter = express.Router();
+apiRouter.use("/contacts", contactsAPIRouter);
+
+contactsAPIRouter.get("/", async (req, res) => {
+  let query = knex.select("*").from("contacts");
+
+  if ("sort" in req.query) {
+    const orderBy = req.query.sort.toString();
+    const check = orderBy.split(" ");
+    console.log(check);
+    if (orderBy.length > 0) {
+      query = query.orderBy(check[0], check[1]);
```

Comment:
you have a great job but still I would prefer to use `knex.raw` with raw parameter binding. As described in [Knex.Js](https://knexjs.org/guide/raw.html). 
In short I would say to do it like: 
`query = query.orderBy(knex.raw('?? ??', [check[0], check[1]]))`